pkg/policies/engine/rego/rego.go

```diff
@@ -43,8 +43,8 @@ const (
 	// EnvironmentModePermissive allows all operations on the compiler
 	EnvironmentModePermissive EnvironmentMode = 1
 	inputArgs                                 = "args"
-	mainRule                                  = "violations"
-	deprecatedMainRule                        = "deny"
+	violationsRule                            = "violations"
```

Comment:
I'd add a comment here saying that violations rule is deprecated in favor of results